src/bounding_boxes.h

```diff
@@ -1,3 +1,17 @@
+/*
+ * OpenSCAD (www.openscad.org)
+ * Copyright 2021 Google LLC
+ *
+ * This program is free software; you can redistribute it and/or
```

Comment:
GPL2-only does not work for OpenSCAD, while the OpenSCAD code itself is GPL2+, CGAL which is a very core part of the application is [GPL3+](https://github.com/CGAL/cgal/blob/master/Installation/LICENSE), so we need the GPL2+ `(or at your option) any later version` definition to build an application that can be distributed.

Reply:
Hi Olivier,

```
 * OpenSCAD (www.openscad.org)
 * Copyright 2021 Google LLC
```
I would also note that this part is a little confusing, since under the standard structure for copyright notices, it looks like Google wrote OpenSCAD (which of course was not your intention).  Of course, Google via you is a contributor now, and it's perfectly reasonable to note that to satisfy your employer's conditions on your flex time.  To prepare ahead for the new files being modified by other contributors after being integrated, and to include the GPL2+ status without being wordy, perhaps a good notice is something like:

```
// Portions of this file are Copyright 2021 Google LLC, and licensed under GPL2+.  See COPYING.
```
Do you think this would satisfy it adequately?

Reply:
Oops good to know, I should have checked, now fixed.

Reply:
@rcolyer sorry, replied before seeing your comment. I'll reword to something less ambiguous indeed!

Reply:
Used your suggestion verbatim :-)